client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -1325,6 +1328,25 @@ public void testClusterPutSettings() throws IOException {
         assertEquals(expectedParams, expectedRequest.getParameters());
     }
 
+    public void testClusterHealth() {
+        ClusterHealthRequest healthRequest = new ClusterHealthRequest();
+        Map<String, String> expectedParams = new HashMap<>();
+        setRandomLocal(healthRequest, expectedParams);
+        setRandomTimeout(healthRequest::timeout, AcknowledgedRequest.DEFAULT_ACK_TIMEOUT, expectedParams);
+        setRandomMasterTimeout(healthRequest, expectedParams);
+        expectedParams.put("level", "shards");
+        // Default value in ClusterHealthRequest is NONE but in Request.Params::withWaitForActiveShards is DEFAULT
```

Comment:
I see, you may have to make changes here like we already did in the 6.x branch where we have a different default for some API.

Reply:
Can you please clarify it? Should I set in ClusterHealthRequest field `waitForActiveShards = ActiveShardCount.DEFAULT`?

Reply:
I mean to adapt the test, doing something along these lines (from the 6.x branch): https://github.com/elastic/elasticsearch/blob/6.x/client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java#L1649 .